DIMS-Core/DIMS-Core.Common/Exceptions/BaseException.cs

```diff
@@ -7,5 +7,8 @@ namespace DIMS_Core.Common.Exceptions
     /// </summary>
     public abstract class BaseException : Exception
     {
+        public BaseException(string message) : base(message)
+        {            
+        }
```

Comment:
You did these changes in this https://github.com/VladPulyak/DIMS-Core.Double-8/pull/5 PR.

Reply:
I made this constructor, because I needed to create custom exception for invalid arguments (Task 3)